backend/YouTubeV2/Controllers/SubscriptionsController.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+using YouTubeV2.Application.DTO;
+using YouTubeV2.Application.Services;
+
+namespace YouTubeV2.Api.Controllers
+{
+	[ApiController]
+	public class SubscriptionsController : ControllerBase
+	{
+		private readonly SubscriptionsService _subscriptionsService;
+
+		public SubscriptionsController(SubscriptionsService subscriptionsService)
+		{
+			_subscriptionsService = subscriptionsService;
+		}
+
+		[HttpGet("subscriptions")]
+		public async Task<ActionResult<IReadOnlyList<SubscriptionDTO>>> GetSubscriptionsAsync([FromQuery][Required] Guid id, CancellationToken cancellationToken)
```

Comment:
I think we need to return SubscriptionsDto which has a list of SubscriptionDto, not just a list. 
![image](https://user-images.githubusercontent.com/95054664/229363526-a3df2e5f-131e-457b-8ed0-6b64316a6a79.png)